static/src/stylesheets/module/experiments/_acquisitions-epic-testimonials.scss

```diff
@@ -0,0 +1,89 @@
+// CSS used for the Epic testimonials test
+
+// Common classes
+
+.epic__testimonial__container {
```

Comment:
All those classnames should follow the [BEM](http://getbem.com/introduction/) naming-convention:

```css
.epic__testimonial__container {}
```

should become

```css
.epic__testimonial-container {}
```

**tl;dr**

- `B` stands for the block. Basically this is your component name (or your class if you think more in the framework of a programming language)
- `E` is the element. Is seperated by `__` from the block. This is just a naming convention, to make it more readable.
- `M` is the modifier. In the best case this is only used on the `B` level (e.g. `epic--testimonial`), to indicate, this is the `epic` component, but with variations. Modifiers are seperated by `--`.